eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -133,6 +139,32 @@ jobs:
             condition: ne(variables['AZURE_RESOURCEGROUP_NAME'], '')
             continueOnError: true
 
+      - script: |
+            npm install
+        workingDirectory: $(Build.SourcesDirectory)/eng/tools/versioning
+        displayName: "Install script dependencies"
+
+      - script: |
+          node getPackagePath.js --repo-root "$(Build.SourcesDirectory)" --package-name "${{parameters.PackageName}}"
+        workingDirectory: $(Build.SourcesDirectory)/eng/tools/versioning
+        displayName: "Get package path"
+
+      - task: PublishCodeCoverageResults@1
+        displayName: 'Publish Code Coverage to DevOps'
+        continueOnError: true
+        condition: and(succeededOrFailed(),eq(variables['TestType'], 'node'),eq(variables['RunCodeCoverage'], true))
+        inputs:
+          codeCoverageTool: Cobertura
+          summaryFileLocation: '$(Build.SourcesDirectory)/$(PackagePath)/coverage/cobertura-coverage.xml'
```

Comment:
Maybe `PackageName` instead of `PackagePath` ?

Reply:
No I think I should remove `Build.SourcesDirectory` from the path